Source/WebCore/page/Page.h

```diff
@@ -555,7 +555,11 @@ class Page : public RefCountedAndCanMakeWeakPtr<Page>, public Supplementable<Pag
     void setTabKeyCyclesThroughElements(bool b) { m_tabKeyCyclesThroughElements = b; }
     bool tabKeyCyclesThroughElements() const { return m_tabKeyCyclesThroughElements; }
 
-    WEBCORE_EXPORT std::optional<FrameIdentifier> findString(const String&, FindOptions, DidWrap* = nullptr);
+    struct FindStringData {
+        std::optional<FrameIdentifier> frameIdentifer;
+        std::optional<SimpleRange> range;
+    };
+    WEBCORE_EXPORT FindStringData findString(const String&, FindOptions, DidWrap* = nullptr);
```

Comment:
since the result is a struct now anyways, the DidWrap out parameter can just be part of the result directly instead

Reply:
Unless we do this: https://github.com/WebKit/WebKit/pull/50265#discussion_r2320699935

Reply:
oops ya, missed that comment

Reply:
I hate an out-param and much as the next person, but this is a thing that is done on many levels of this code, I or someone else should fix this as a followup.